packages/@react-aria/steplist/src/useStepListItem.ts

```diff
@@ -51,7 +51,7 @@ export function useStepListItem<T>(props: AriaStepListItemProps, state: StepList
       event.stopPropagation();
     }
 
-    itemProps.onKeyDown(event);
+    itemProps?.onKeyDown?.(event);
```

Comment:
Not an issue to be over cautions, but itemProps won't be null so you don't need the first '?'.